arm/examples/create/create.go

```diff
@@ -1,16 +1,21 @@
-package create
+package main
 
 import (
 	"fmt"
 	"log"
+	"math/rand"
+	"time"
 
 	"github.com/Azure/azure-sdk-for-go/arm/examples/helpers"
 	"github.com/Azure/azure-sdk-for-go/arm/storage"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-func createAccount(resourceGroup, name string) {
+func main() {
+	resourceGroup := "resourceGroupName"
+	name := fmt.Sprintf("gosdktestname%s", randomString(5))
+
 	c, err := helpers.LoadCredentials()
```

Comment:
Just a note, there's a "complete" auth example, but I put it in go-autorest since that's where auth is implemented: https://github.com/Azure/go-autorest/tree/master/autorest/azure/example